supported/osg-htc/osg-hosted-ce/templates/configmap.yaml

```diff
@@ -138,17 +138,14 @@ data:
     JOB_ROUTER_POST_ROUTE_TRANSFORM_NAMES = $(JOB_ROUTER_POST_ROUTE_TRANSFORM_NAMES) Token
 
     JOB_ROUTER_TRANSFORM_GridResource @=jrt
-        gr = batch {{ .Values.RemoteCluster.Batch }} $(MY.Owner)@{{ .Values.RemoteCluster.LoginHost }}
-        gr = $(gr) --rgahp-glite ~/{{ .Values.RemoteCluster.BoscoDir | default "~/bosco"}}/glite
-        {{ if .Values.BoscoOverrides.TarballURL | default "" | contains "bosco-1.3"  }}
-        gr = $(gr) --rgahp-script batch_gahp
-        {{ end }}
-        {{ if not .Values.RemoteCluster.SSHBatchMode }}
-        gr = $(gr) --rgahp-nobatchmode
-        {{ end }}
-        {{ if not .Values.RemoteCluster.LoginShell }}
-        gr = $(gr) --rgahp-nologin
-        {{ end }}
+        hosted_grid_resource = batch {{ .Values.RemoteCluster.Batch }} $(MY.Owner)@{{ .Values.RemoteCluster.LoginHost }} --rgahp-glite ~/{{ .Values.RemoteCluster.BoscoDir | default "~/bosco"}}/glite
+        {{- if .Values.BoscoOverrides.TarballURL | default "" | contains "bosco-1.3" }} --rgahp-script batch_gahp{{ end }}
```

Comment:
```suggestion
```

I think we can actually drop this one. There don't appear to be any CEs left that are using such an old Bosco tarball

Reply:
Ah, the change here is about a specific ancient bosco tarball.